src/main/java/com/google/sps/data/FirestoreDB.java

```diff
@@ -134,6 +134,16 @@ public void deleteUserProperty(String userId, String objectId, String collection
     DBUtils.blockOnFuture(removeUserPropertyFuture);
   }
 
+  public Boolean inUserMap(String userId, String recipeId, String mapName) {
+    DocumentSnapshot user = DBUtils.blockOnFuture(DBUtils.user(userId).get());
+
+    if (!user.exists()) {
+      return false;
```

Comment:
According to the javadoc it should return null when user doesn't exist?

Reply:
Good catch.